common/governments_and_reforms.cwt

```diff
@@ -673,6 +662,24 @@ government_reform = {
 				## cardinality = 0..1
 				mercs_do_not_cost_army_professionalism = bool
 			}
+		
+			## cardinality = 0..1
+			nomad = bool
+			## cardinality = 0..1
+			native_mechanic = bool
+			## cardinality = 0..1
+			allow_force_tributary = bool
```

Comment:
Would it be possible to add a comment to these (allow_force_tributary, boost_income, is_trading_city) so it is clear to the modders?

use the `###Comment Here, start it with no space`

Example for a modifier with the comment, as rendered in VSCode

![image](https://github.com/cwtools/cwtools-eu4-config/assets/30738253/d1a599f7-2b0d-41f6-8eaf-3821c0c8a764)